.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -128,8 +128,10 @@ steps:
 
   - group: "Filebeat Extended Tests"
     key: "filebeat-extended-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*(macOS|arm).*/
```

Comment:
we shouldn't be running macos tests with merge commits, see https://github.com/elastic/beats/pull/39533/files
therefore we should remove

```
build.env("BUILDKITE_PULL_REQUEST") == "false"
```

Reply:
as discussed we will keep this to mirror `main`

Reply:
Kept, since same on main